src/rules/mergers.ts

```diff
@@ -1,15 +1,16 @@
 import { mergeBanTypes } from "./mergers/ban-types";
 import { mergeIndent } from "./mergers/indent";
 import { mergeNoMemberDelimiterStyle } from "./mergers/member-delimiter-style";
-import { mergeNoCaller } from "./mergers/no-caller";
+import { mergeRuleWithNoOptions } from "./mergers/rule-with-no-options";
 import { mergeNoEval } from "./mergers/no-eval";
 import { mergeNoUnnecessaryTypeAssertion } from "./mergers/no-unnecessary-type-assertion";
 
 export const mergers = new Map([
     ["@typescript-eslint/ban-types", mergeBanTypes],
+    ["@typescript-eslint/consistent-type-assertions", mergeRuleWithNoOptions],
```

Comment:
Oh, but [`@typescript-eslint/consistent-type-assertions`](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/consistent-type-assertions.md) does actually have options. If one of the configurations being merged has options, they'll need to be kept.